src/Mailer/Message/AdherentAccountActivationMessage.php

```diff
@@ -1,19 +1,19 @@
 <?php
 
-namespace AppBundle\Mailjet\Message;
+namespace AppBundle\Mailer\Message;
 
 use AppBundle\Entity\Adherent;
 use Ramsey\Uuid\Uuid;
 
-final class AdherentAccountActivationMessage extends MailjetMessage
+final class AdherentAccountActivationMessage extends Message
 {
     public static function createFromAdherent(Adherent $adherent, string $confirmationLink): self
     {
         return new self(
             Uuid::uuid4(),
             '54665',
             $adherent->getEmailAddress(),
-            self::fixMailjetParsing($adherent->getFullName()),
```

Comment:
Why removing this?

Reply:
i moved this logic inside the AppBundle\Mailjet\EmailTemplate class.
It's Mailjet specific now!